.github/workflows/pr-source-check.yaml

```
@@ -0,0 +1,17 @@
+
+on:
+  pull_request:
+    branches: [main]
+
+jobs:
+  pr-source-check:
+    name: PR source must be develop
+    runs-on: ubuntu-latest
+    steps:
+      - name: Ensure PR source is develop
+        run: |
```

actionlint (1.7.12)

[error] "github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)